### PR TITLE
Elle Frankenberg- news site

### PR DESCRIPTION
**Pull request by ElleFrankenberg**

### CODE REVIEW

**Comment by JennieDalgren** [comment]
Hi! Do you have a deployed version of this? Please provide a link to the project 🙂

**Comment by ElleFrankenberg** [comment]
> Hi! Do you have a deployed version of this? Please provide a link to the project 🙂

Hi! Here is the link :) https://quizzical-fermi-345eed.netlify.app

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+If I had more time, I would have liked to try to add some animation to my project. I'm not sure if the page is responsive now, but I hope so, but I would have liked to be completely sure. I still have not understood how to make the pictures and boxes stop growing.
 
 ## View it live
 Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
```
Comment (JennieDalgren) [inline README.md:12]: this is where you should add your link, for future projects

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,113 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Krukväxter</title>
     <link rel="stylesheet" href="./style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
```
Comment (JennieDalgren) [inline code/index.html:9]: ⭐

File: code/index.html [hunk 3]
```diff
@@ -2,21 +2,113 @@
...
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
+    <header class="header">
       <!-- Some kind of logo and navbar -->
+      <img class="logo" src="./potplantlogo.png" alt="logo"/>
+      <h1 class="titel">Krukväxter</h1>
+      <nav class="navigation-hamburger-container">
+        <div class="hamburger">
```
Comment (JennieDalgren) [inline code/index.html:18]: seems like this div doesnt have a closing tag?

File: code/index.html [hunk 4]
```diff
@@ -2,21 +2,113 @@
...
+        <a class="links href="">Skötsel</a>
+        <a class="links href="">Blomsteraffärer</a>
+        <a class="links href="">Om</a>
+        <a class="links href="">Kontakt</a>
+      </nav> 
     </header>
+    
+    
+    <section class="main-plant-container">
+      <div class="main-plant-artikle">
```
Comment (JennieDalgren) [inline code/index.html:33]: seems like you have a section with the classname _main plant container_ but then just a div with the class name _main plant artikle_ in that. You could just keep one of them, preferable the section as it is a more semnatic element than a div.

File: code/index.html [hunk 5]
```diff
@@ -2,21 +2,113 @@
...
+ </section>
+
+  <div class="other-plant-title">
+    <h2>OLIKA SORTERS KRUKVÄXTER</h2>
+    <p>Här är några fler sorters krukväxter</p>
+  </div>
+
+    <section class="other-plant-container"> 
+      <section class="container">
+      <section class="other-plant-cards">
```
Comment (JennieDalgren) [inline code/index.html:53]: again here with sections...
Do they all have closing tags?

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,435 @@
...
+}
+
+.other-plant-img {
+  width: 100%;
+}
+
+
+.other-plant-container h2 {
+background-color: azure;
+margin: 20px 0px 5px 0px;
```
Comment (JennieDalgren) [inline code/style.css:237]: indentation

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,435 @@
...
+margin: 30px 0px 20px 0px;
+}
+
+/* FOOTER */
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
```
Comment (JennieDalgren) [inline code/style.css:250]: remove blank line

**Review by JennieDalgren — CHANGES_REQUESTED** [review]
Good job with your project.

For next time, try to commit more often and with descriptive commit messages.

To be able to approve this project you should fix the indentation in your code.

Take a look at [the code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you need and change this.
We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

really nice job with the styling, I like the design!
keep it up ⭐️

**Comment by ElleFrankenberg** [comment]
Hi! Now I have fixed the required changes :) (I hope)

**Comment by JennieDalgren** [comment]
Hey, I can see that you've made changes but I can't access the html or css files any more. Are you sure they are still there, in your repo?

**Comment by ElleFrankenberg** [comment]
Hi! now I believe the html and css is in the repo (I hope)

**Review by JennieDalgren — APPROVED** [review]
💯

**Pull request closed without merging**